app/Platform/Commands/NoIntroImport.php

```diff
@@ -13,7 +13,7 @@
 
 class NoIntroImport extends Command
 {
-    protected $signature = 'ra:server:rom:no-intro:import {jsonDatFile} {systemId?} {--seed} {--ignore-system-mismatch}';
+    protected $signature = 'ra:platform:rom:no-intro:import {jsonDatFile} {systemId?} {--seed} {--ignore-system-mismatch}';
```

Comment:
The `ra:server` namespace for commands is a leftover from before it was called `platform`.